### PR TITLE
Issue 115 merge private SubstationCost into public SubstationCost module

### PR DESCRIPTION
**Pull request by parangat94**

### CODE REVIEW

**Comment by parangat94** [comment]
Parangat is currently reviewing this.

**Comment by parangat94** [comment]
@akey7 @eberlea ready for review.

File: landbosse/tests/model/test_SubstationCost.py [hunk 1]
```diff
@@ -10,7 +10,11 @@ class TestSubstationCost(TestCase):
     def setUp(self):
```
Comment (eberlea) [inline landbosse/tests/model/test_SubstationCost.py]: Please remove chances to the test data before you merge.

Comment (parangat94) [reply landbosse/tests/model/test_SubstationCost.py]: Hi @eberlea, just so I'm on the same page, would you like me to revert code in the test_SubstationCost.py to a version that sits in the develop branch?

Comment (eberlea) [reply landbosse/tests/model/test_SubstationCost.py]: Ah, if the changes are in the develop branch, then just leave them be. If you're making the changes in your branch, then please revert.

**Comment by akey7** [comment]
@parangat94 Has there been resolution on the test file issue?

**Pull request merged**